src/scripts/seqbox_utils.py

```diff
@@ -323,14 +354,16 @@ def read_in_sample_source_info(sample_source_info):
 def read_in_culture(culture_info):
     check_cultures(culture_info)
     culture = Culture()
-    culture.date_cultured = culture_info['date_cultured']
```

Comment:
in some places you're changing from e.g.  from `culture.date_cultured = culture_info['date_cultured']` to `culture.date_cultured = datetime.datetime.strptime(culture_info['date_cultured'],'%d/%m/%Y')`, in other places you're changing from `extraction.date_extracted = datetime.datetime.strptime(extraction_info['date_extracted'], '%d/%m/%Y')` to `extraction.date_extracted = extraction_info['date_extracted']`.

What determines which way round you are changing? i.e. why do you sometimes use datetime.datetime.strptime and sometimes not use it?

Reply:
Problem: that if the culture info comes from an excel file, this line will break because culture_info['date_cultured'] will already be a datetime obj.

Solution: Do all the conversions to datetime objs in the read_csv/read_excel functions.

Documentation: all dates in the input file have to contain the word "date" and nothing that we don't want to be parsed as a date should contain the word "date"

Reply:
Also, replace NaNs with None in the read_excel function as well. Remove calls to this function elsewhere in the code.